scam-detector-py/src/agent.py

```diff
@@ -1204,7 +1204,8 @@ def detect_scammer_contract_creation(w3, transaction_event: forta_agent.transact
             logging.info(f"{BOT_VERSION}: {transaction_event.from_} created contract {created_contract_address}")
             original_threat_category, original_alert_hash = get_original_threat_category_alert_hash(transaction_event.from_)
             if original_threat_category != "":
-                findings.append(ScamDetectorFinding.scammer_contract_deployment(transaction_event.from_, created_contract_address.lower(), original_threat_category, original_alert_hash, CHAIN_ID))
+                future_contract_addresses = [Utils.calc_contract_address(w3, transaction_event.from_, n).lower() for n in range(nonce + 1, nonce + 11)]
```

Comment:
I dont think we want this here because then we add a lot of labels even when the contract creation happens through usual means. I havent seen contract addresses used this way in scams. Have you?

Reply:
My thinking was that if a scammer deploys a contract (even directly), it would be possible that more scam contracts could be deployed. However, I get that this will create a lot of redundant labels, so I removed it.